amf-api-contract/shared/src/main/scala/amf/apicontract/client/common/ProvidedMediaType.scala

```diff
@@ -8,6 +8,8 @@ import scala.scalajs.js.annotation.{JSExportAll, JSExportTopLevel}
 @JSExportAll
 object ProvidedMediaType {
 
+  def build(mediaType: String, syntax: String): String = mediaType + syntax
```

Comment:
this will not build a valid mediaType, it is missing the '+' sign.

Reply:
Isn't this just Vendor + Syntax? Media types already include syntax

Reply:
Strictly speaking we should use these media types (not registered in IANA): https://en.wikipedia.org/wiki/Media_type#Unregistered_tree

Reply:
Yup, agree on this one. Will fix it.

Reply:
Dropped method as these media types will no longer be used. Change pending